terra_opp/models.py

```diff
@@ -169,7 +177,7 @@ class Picture(BaseUpdatableModel):
     )
 
     properties = JSONField(_("Properties"), default=dict, blank=True)
-    file = VersatileImageField(_("File"))
+    file = VersatileImageField(_("File"), upload_to=image_upload_to)
```

Comment:
> upload_to may also be a callable, such as a function. This will be called to obtain the upload path, including the filename. This callable must accept two arguments and return a Unix-style path (with forward slashes) to be passed along to the storage system
cf: https://docs.djangoproject.com/en/3.1/ref/models/fields/#filefield